rsts/community/troubleshoot.rst

```diff
@@ -121,6 +121,25 @@ Troubles with handling large responses in ``FlyteRemote.sync``
 - To fix this, edit the ``flyte-admin-base-config`` config map using the command ``kubectl edit cm flyte-admin-base-config -n flyte`` to increase the ``maxMessageSizeBytes`` value.
 
 
+Issues with long running workflow executions (more than an hour)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Flyte users may see this error when long-running workflow executions (typically after an hour) are returned by flytekit:
+
+    - `flytekit.exceptions.scopes.FlyteScopedUserException: An error occurred (ExpiredToken) when calling the AssumeRole operation: The security token included in the request is expired`
```

Comment:
Is this true for all cases like using oidc (eks Iam for service accounts?)

Reply:
I believe so, although I'm not sure what are the default values in gcp.